ansible/ansible-playbook.yml

```
@@ -0,0 +1,110 @@
+- name: Update web servers
+  hosts: managed-nodes
+  remote_user: ec2-user
+  vars:
+    host: "{{ inventory_hostname }}"
+  vars_files:
+    - .ansible-secrets.yml
+
+  tasks:
+    - name: Install Git
+      yum:
+        name: "git"
+        state: "present"
+        update_cache: yes
+      become: yes # Installing git must be run as the root user
+
+    - name: Install Python
+      yum:
+        name: "python3.11"
+        state: "present"
+        update_cache: yes
+      become: yes # Installing python must be run as the root user
+
+    - name: Install Node
+      yum:
+        name: "nodejs"
+        state: "present"
+        update_cache: yes
+      become: yes # Installing node must be run as the root user
```

Minor, but worth noting that you can combine these:
```suggestion
    - name: Install Git, Python, Node
      yum:
        name: ["git", "python3.11", "nodejs"]
        state: "present"
        update_cache: yes
      become: yes # Installing through yum must be run as the root user
```

---

Ah, that's much cleaner!